com.unity.netcode.gameobjects/Components/NetworkRigidbody.cs

```diff
@@ -5,77 +5,96 @@ namespace Unity.Netcode.Components
 {
     /// <summary>
     /// NetworkRigidbody allows for the use of <see cref="Rigidbody"/> on network objects. By controlling the kinematic
-    /// mode of the rigidbody and disabling it on all peers but the authoritative one.
+    /// mode of the <see cref="Rigidbody"/> and disabling it on all peers but the authoritative one.
     /// </summary>
     [RequireComponent(typeof(Rigidbody))]
     [RequireComponent(typeof(NetworkTransform))]
     public class NetworkRigidbody : NetworkBehaviour
     {
+        /// <summary>
+        /// Determines if we are server (true) or owner (false) authoritative
+        /// <see cref="NetworkTransform.IsServerAuthoritative"/>
+        /// </summary>
```

Comment:
Are doc comments good practice for private members?

Reply:
Nice catch!
(removed)